nekoyume/Assets/Planetarium/Nekoyume/Editor/ResourceEditor.cs

```diff
@@ -0,0 +1,43 @@
+using System.Text;
+using Nekoyume.Game.VFX.Skill;
+using UnityEditor;
+using UnityEngine;
+
+public static class ResourceEditor
+{
+    private const string VFXPrefabPath = "Assets/Resources/VFX/";
+
+    [MenuItem("Tools/Resource/Set All Character VFX Prefab Rendering Layer")]
+    public static void SetAllCharacterVFXPrefabRenderingLayer()
+    {
+        // TODO: 일단 현재 프로젝트의 리소스 구조대로 스크립트를 짜긴 했는데, 리소스가 구분이 안되는 느낌이라 경로부터 수정해야할 것 같음
+        var vfxAssetPath = AssetDatabase.FindAssets("t:GameObject", new[] { VFXPrefabPath });
+
+        var sb = new StringBuilder();
+        sb.AppendLine("Set Character VFX Prefab Rendering Layer:");
+        foreach (var assetPath in vfxAssetPath)
+        {
+            var asset = AssetDatabase.LoadAssetAtPath<GameObject>(AssetDatabase.GUIDToAssetPath(assetPath));
+            if (asset == null)
+            {
+                continue;
+            }
+
+            // SkillVFX & BuffVFX 상대로만 레이어 처리
+            if (!asset.TryGetComponent<SkillVFX>(out _) &&
```

Comment:
이왕 하는김에 VFX 상속받는 스크립트를 모두 적용해주는게 좋을것같아보입니다~!
```suggestion
            if (!asset.TryGetComponent<SkillVFX>(out _) &&
```

Reply:
엇~!  그렇군요 ㄷㄷㄷ  확인감사합니다~!